doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -90,6 +90,11 @@ Bluetooth® LE
   The previous ``hci_err.h`` header has been merged into the new ``hci_types.h`` header.
   This can break builds that were directly including ``hci_err.h``.
 
+* Added:
+
+  * :ref:`esl_service_readme` library.
+  * :ref:`esl_service_client_readme` library.
```

Comment:
You should remove those since they're not usable outside of the esl application now

Reply:
Remove